lib/mp4_decode_bs_impl.cc

```diff
@@ -214,11 +214,11 @@ namespace gr {
         baudRate = sample_rate;
       }
       d_sample_rate = sample_rate;
-      GR_LOG_DEBUG(d_logger, format("bytes consumed %d") % (int) (hInfo.bytesconsumed));
-      GR_LOG_DEBUG(d_logger,
+      //GR_LOG_DEBUG(d_logger, format("bytes consumed %d") % (int) (hInfo.bytesconsumed));
+      /*GR_LOG_DEBUG(d_logger,
```

Comment:
Same as above: GR_LOG_DEBUG should be 0 overhead when not compiled with debug flags. so don't comment it out.